specification/webhooks.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: Providing Webhooks
+parent: The OpenAPI Specification Explained
+nav_order: 9
+---
+
+# Providing Webhooks
+
+Webhooks are a style of communication common in the API economy. Their purpose is to provide bi-directional communications from the API provider to the API consumer to support out-of-band events that do not fit into the pattern of REST APIs. Webhooks fulfill similar purposes to Callbacks in that they support transmitting information about asynchronous or long-running communications. The example of payments APIs discussed in the Callbacks topic also provides an example of the real-world implementations of Webhooks. Many payments API providers implement webhooks to communicate the result of payment operations as an alternative to callbacks. API consumers can implement a webhook at a URL of their choosing to receive out-of-band updates on a previously submitted, long-running request. This avoids implementing polling patterns, which many API providers find onerous to support.
```

Comment:
should we distinguish that with the callback that usually in the webhook the client is not the initiator of the request. like i can subscribe to the github webhook of commit , so that even if i do not do a commit but someone else i got notified 

and that webhook suppose a pre registration

Reply:
I've added clarifications here and used the example of GitHub to show how pre-registration can facilitate using Webhooks.